src/main/java/core/basesyntax/FibonacciSequence.java

```diff
@@ -10,6 +10,12 @@
      * Результат: 21</p>
      */
     public int getFibonacciNumber(int index) {
-        return 0;
+        if (index == 0) {
+            return 0;
+        }
+        if (index == 1 || index == -1) {
```

Comment:
why do we need to check for negative indices?

Reply:
done